cypress/integration/happy-path.spec.js

```diff
@@ -1,7 +1,7 @@
 describe('Happy path testing', function() {
   it('Should complete happy path successfully from scope diagnosis to confirmation page', function () {
 
-    cy.visit("https://lga-1543-install-and-integra.staging.checklegalaid.service.gov.uk/start")
+    cy.visit("https://lga-1509-test-happy-path-wit.staging.checklegalaid.service.gov.uk/start")
```

Comment:
Why is this hard-coded? Is this going to only run on master branch, and therefore staging url?
I had assumed it would be used on feature branches, so would need to use the UAT url dynamically

Reply:
Yeah this is one of the bugs I need to fix. As part of the ticket, I will make this dynamic by using environmental variables and running the command in circleci with the correct cli arguments

Reply:
OK